bundle/README.md

```diff
@@ -71,6 +71,36 @@ For a list of available configuration for the `values` field, refer to [Istio's
 - [CNI parameters](https://artifacthub.io/packages/helm/istio-official/cni?modal=values)
 - [ZTunnel parameters](https://artifacthub.io/packages/helm/istio-official/ztunnel?modal=values)
 
+## Installing the istioctl tool
+
+The istioctl tool is a configuration command line utility that allows service operators to debug and diagnose Istio service mesh deployments.
+
+### Prerequisites
+
+Use an istioctl version that is the same version as the Istio control plane for the Service Mesh deployment. See [Istio Releases](https://github.com/istio/istio/releases) for a list of valid releases, including Beta releases.
+
+### Procedure
+
+1. Confirm if you have istioctl installed, and if so which version, by running the following command at the terminal:
+
+  ```sh
+    $ istioctl version
```

Comment:
I think we should verify the version of Istio the user is running with `oc -n istio-system get istio`

Reply:
Thank you, this was one of the questions I was going to ask

Reply:
Done